### PR TITLE
add check for NaN for diff

### PR DESCRIPTION
**Pull request by pedroha**
Fixes autosize when padding is not defined at all (i.e. diff was NaN)

### CODE REVIEW

**Comment by javierjulio** [comment]
@pedroha thanks! Sorry I couldn't get this in yesterday. Just have one request I'll comment on the line of code.

File: dist/jquery.textarea_autosize.js [hunk 1]
```diff
@@ -24,10 +24,14 @@
       var diff = parseInt(this.$element.css('paddingBottom')) +
                   parseInt(this.$element.css('paddingTop'));
 
+      diff = (diff) ? diff: 0;
+
       if (containsText(this.element.value)) {
         this.$element.height(this.element.scrollHeight - diff);
       }
 
+      // this.$element.css('outline', '1px solid red');
```
Comment (javierjulio) [inline dist/jquery.textarea_autosize.js]: @pedroha can you remove the commented lines 33,34 from the dist file? That's a generated file.

File: dist/jquery.textarea_autosize.js [hunk 2]
```diff
@@ -24,10 +24,14 @@
       var diff = parseInt(this.$element.css('paddingBottom')) +
                   parseInt(this.$element.css('paddingTop'));
 
+      diff = (diff) ? diff: 0;
```
Comment (javierjulio) [inline dist/jquery.textarea_autosize.js]: @pedroha can you please remove this line entirely and instead default diff to 0:

```js
var diff = parseInt(this.$element.css('paddingBottom')) + parseInt(this.$element.css('paddingTop')) || 0;
```

**Comment by pedroha** [comment]
No worries (likewise I was busy). I tested and included your revised check for NaN.

Cheers!
Pedro

**Comment by javierjulio** [comment]
Awesome thanks @pedroha! 🙏🏻❤️

**Comment by javierjulio** [comment]
Just released 0.4.1. Let me know if you run into any issues @pedroha thanks!

**Comment by pedroha** [comment]
No issues at all (well, I tested it before submitting the revised PR).

FYI. I noticed that I have to add an extra <div> for Safari when the textarea is floated (so that it doesn't disappear when I press return). That extra <div> fixed it but it was unstyled and was giving me the NaN edge case (which is now fixed :)

Hasta luego,
Pedro

**Pull request merged**